src/App.tsx

```
@@ -1,10 +1,106 @@
+      if (user.status === "active") {
+        if (user.verified === true) {
+          result = true;
+        }
+      }
+    }
+  }
+  return result;
+}
+
+// ❌ Functions not following naming conventions
+function getData() {
```

ESLint

[error] 'getData' is defined but never used.

(@typescript-eslint/no-unused-vars)